packages/react/src/components/F0Card/CardInternal.tsx

```diff
@@ -108,6 +108,12 @@ export interface CardInternalProps {
    * Whether the card should have a full height
    */
   fullHeight?: boolean
+
+  /**
+   * When true, disables the full-card overlay link so parent components
+   * can manage drag-and-drop while still allowing click navigation via onClick
+   */
+  disableOverlayLink?: boolean
```

Comment:
add this property as private in the F0Card.tsx to avoid using it from outside f0